cpp/include/cudf/strings/convert/convert_lists.hpp

```diff
@@ -50,17 +50,19 @@ namespace strings {
  *
  * @throw cudf::logic_error if the input column is not a LIST type with a STRING child.
  *
- * @param input Lists column to format.
- * @param na_rep Replacement string for null elements.
- * @param separators Strings to use for enclosing list components and separating elements.
- * @param mr Device memory resource used to allocate the returned column's device memory.
- * @return New strings column.
+ * @param input Lists column to format
+ * @param na_rep Replacement string for null elements
+ * @param separators Strings to use for enclosing list components and separating elements
+ * @param stream CUDA stream used for device memory operations and kernel launches
+ * @param mr Device memory resource used to allocate the returned column's device memory
+ * @return New strings column
  */
 std::unique_ptr<column> format_list_column(
   lists_column_view const& input,
-  string_scalar const& na_rep           = string_scalar("NULL"),
+  string_scalar const& na_rep           = string_scalar(""),
```

Comment:
This is a change in behaviour.  Was this change intentional?  On the face of it, this seems unrelated to the task at hand.

Reply:
This was intentional. The 'NULL' string-scalar requires allocating device memory using the default stream which will fail the stream tests. An empty string does not require allocating and device memory. One of the many reasons this PR is marked as breaking.

Reply:
P.S. Should we be removing this default?  I see that the default parameters have been removed in other locations.

Reply:
Ah, thanks for clarifying. That makes sense.

Reply:
I see no reason to remove the new default.